lib/apiservers/engine/backends/container.go

```diff
@@ -142,8 +150,8 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 	var err error
 
 	//TODO: validate the config parameters
-	log.Debugf("config.Config = %+v", config.Config)
-
+	log.Printf("Image fetch section - Container Create")
```

Comment:
What does this message actually mean?  I.e. is it useful in debugging?  Can it be made clearer?